teuthology/orchestra/remote.py

```diff
@@ -206,6 +206,23 @@ def run(self, **kwargs):
         r.remote = self
         return r
 
+    def mkdtemp(self, suffix=None, parentdir=None):
+        """
+        Create a temporary directory on remote machine and return it's path.
+        """
+        args = ['mktemp', '-d']
+
+        if suffix is not None:
```

Comment:
(style) it will be more pythonic if you use short check:
```
if suffix:
```

Reply:
Actually. :P

Thanks!